test-utils/org.wso2.carbon.identity.testutil/pom.xml

```diff
@@ -51,6 +51,18 @@
             <artifactId>powermock-module-testng-common</artifactId>
             <scope>provided</scope>
         </dependency>
+        
+        <!-- MSF4J currently used only on tests, and dependency is added at this project only, not at root pom -->
+        <dependency>
+            <groupId>org.wso2.msf4j</groupId>
+            <artifactId>msf4j-core</artifactId>
+            <version>${msf4j.version}</version>
```

Comment:
shall we add the dependency version to the dependency management in the parent pom and remove from here?